classifai.php

```diff
@@ -88,7 +88,7 @@ function classifai_autoload() {
 		return true;
 	} else {
 		error_log( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
-			sprintf( esc_html__( 'Fatal Error: Composer not setup in %', 'classifai' ), CLASSIFAI_PLUGIN_DIR )
+			sprintf( 'Fatal Error: Composer not setup in %s', CLASSIFAI_PLUGIN_DIR )
```

Comment:
```suggestion
			sprintf( 'Warning: Composer not setup in %s', CLASSIFAI_PLUGIN_DIR )
```

Or maybe notice, the error isn't fatal as Classifai catches it.